schema/model_pointset_point.json

```diff
@@ -42,6 +42,16 @@
       "description": "Mapping for the point to an internal resource (e.g. BACnet object reference)",
       "type": "string",
       "examples": ["AI3", "400070"]
+    },
+    "translation": {
+      "description": "An ontological translation for the point",
+      "type": "string",
+      "examples": ["return_air_temperature_sensor"]
+    },
+    "tags": {
+      "description": "Descriptor for the point",
+      "type": "string",
+      "examples": ["lighting", "lighting,building"]
```

Comment:
Array of strings seems better than a singular CSV string?  Also consider a (restricted) regex for this (e.g. only [a-z] for actual tag names)... ?  I generally fret about fields like this because they end up getting conscripted to do nefarious things.  Also, what's the future proofing here?  singular string tags are nice but also very limiting, and typically the NEXT thing that people want to do is key=value pairs, so the question comes down to "should we just start with key=value pairs and skip over tags completely" -- rather than support two structures in the future.

Reply:
after our quick chat I understand "why not k=v", but then the question remains about "why a CSV rather than a list/array"

Reply:
Changed into a list of alphanumeric string values